flink-dist/src/main/flink-bin/bin/jobmanager.sh

```diff
@@ -51,6 +51,10 @@ if [[ $STARTSTOP == "start" ]] || [[ $STARTSTOP == "start-foreground" ]]; then
         args+=("--webui-port")
         args+=("${WEBUIPORT}")
     fi
+
+    if [ ! -z "${DYNAMIC_PARAMETERS+x}" ]; then
```

Comment:
Why is a `+x` appended to `DYNAMIC_PARAMETERS`? In the other scripts this suffix is missing.

Reply:
True I forgot to align all of that. The variable extension is not necessary in our case. Hence, I removed it again aligning all three code snippets.